anta/runner.py

```diff
@@ -41,6 +41,14 @@ async def main(
         any: ResultManager object gets updated with the test results.
     """
 
+    if not tests:
+        logger.info("The list of tests is empty, exiting")
+        return
+
+    if len(inventory) == 0:
+        logger.info("The inventory is empty, exiting")
+        return
```

Comment:
I think it would be better to check after we call the `connect_inventory()` and `get_inventory()` methods on the inventory. 

That will cover the situation where the inventory has a single device and there is a connection problem caught by `connect_inventory()`.

Reply:
I think that should be a different log

Reply:
Agreed.